tools/package_tools/mmdeploy_builder.py

```diff
@@ -56,7 +56,14 @@ def _call_command(cmd, cwd, stdout=None, stderr=None):
         return
     logging.info(f'Process cmd: {cmd}')
     logging.info(f'work_path: {cwd}')
-    run(cmd, stdout=stdout, stderr=stderr, cwd=cwd, shell=True)
+    try:
+        ret = run(cmd, stdout=stdout, stderr=stderr, cwd=cwd, shell=True)
+        if ret.returncode != 0:
+            logging.error(f'Process cmd: "{cmd}"'
+                          f' failed with returncode: {ret.returncode}')
+    except Exception:
```

Comment:
error from subprocess cannot be catched here.

Reply:
Is it possible to catch the exception throw from the subprocess of commandline?

Reply:
I've tested the try-catch won't work when building fails.

Reply:
Errr, I means it is impossible to catch an exception from `subprocess.run`.

Reply:
OK